modules/delegates/helpers/filterDelegates.ts

```diff
@@ -59,3 +61,41 @@ export function filterDelegates(
       })
   );
 }
+
+export function filterDelegateAddresses(
+  githubDelegates: DelegateRepoInformation[] | undefined,
+  allDelegateAddresses: AllDelegatesEntry[],
```

Comment:
wouldn't it be better to already have joined the data? 

I believe there's an entity called Delegate, (you can also use a partial delegate) instead of sending 2 different arrays. 

You would just need to filter over 1 entity.

Reply:
Also adding some unit tests for the filter function could be good

Reply:
I'll be adding the unit tests tomorrow